### PR TITLE
Update code-generator for new apiVersion

### PR DESCRIPTION
**Pull request by zhengxiexie**

### CODE REVIEW

**Review by dantingl — DISMISSED** [review]

File: pkg/client/clientset/versioned/typed/nsx.vmware.com/v1alpha1/nsxserviceaccount.go [hunk 1]
```diff
@@ -1,182 +0,0 @@
-/* Copyright © 2024 VMware, Inc. All Rights Reserved.
```
Comment (gran-vmv) [inline pkg/client/clientset/versioned/typed/nsx.vmware.com/v1alpha1/nsxserviceaccount.go:1]: Is this file no longer needed? Who is the consumer of this clientset/informer/lister?

**Review by dantingl — APPROVED** [review]

File: pkg/controllers/subnetset/subnetset_webhook.go [hunk 2]
```diff
@@ -29,7 +29,7 @@ var NSXOperatorSA = "system:serviceaccount:vmware-system-nsx:ncp-svc-account"
 
 type SubnetSetValidator struct {
 	Client  client.Client
-	decoder *admission.Decoder
+	decoder admission.Decoder
```
Comment (timdengyun) [inline pkg/controllers/subnetset/subnetset_webhook.go:32]: why need change this file?

Comment (zhengxiexie) [reply pkg/controllers/subnetset/subnetset_webhook.go:32]: When building
go: toolchain upgrade needed to resolve k8s.io/client-go/discovery
go: k8s.io/client-go@v0.30.3 requires go >= 1.22.0; switching to go1.22.5

And then controller-runtime/pkg/webhook/admission is not compatible with the previous, so I updated it.

**Review by timdengyun — APPROVED** [review]

**Pull request merged**